boards/arm/nrf54l15pdk_nrf54l15/nrf54l15pdk_nrf54l15_cpuapp_ns_defconfig

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+CONFIG_SOC_SERIES_NRF54LX=y
+CONFIG_SOC_NRF54L15_ENGA_CPUAPP=y
+CONFIG_BOARD_NRF54L15PDK_NRF54L15_CPUAPP_NS=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# MPU-based null-pointer dereferencing detection cannot
+# be applied as the (0x0 - 0x400) is unmapped for this target.
```

Comment:
This isn't true, is it?

When I look at the datasheet it looks like RRAM starts at address 0.

Reply:
Fair point, this was actually copy pasted by the defconfig file in Zephyr.  I also don't see how this can apply for this device so I will remove.